unkeyed/unkey
# feat: option to override .env file

## What does this PR do?

Adds option to override the `.env` when bootstrapping the dashboard app.

Fixes #1635 



## Type of change




## How should this be tested?



- Run `pnpm local` and select dashboard, after entering the required values, a `.env` file should be created.
- Rerun the above command and input the necessary prompts. You should see a message saying the `.env` file already exists, do you want to override it? If you click yes, you should see the `.env` file updated with the latest values, else you should see a log message in the terminal with the variable values that should be added to the `.env` file.

## Checklist



### Required


### Appreciated
